src/content/docs/mobile-monitoring/new-relic-monitoring-react-native/react-native-installation.mdx

```diff
@@ -0,0 +1,312 @@
+---
+title: React Native agent
+tags:
+  - Mobile monitoring
+  - New Relic Mobile
+  - Configuration
+metaDescription: Instructions for installing and configuring the New Relic React Native agent.
+---
+
+With our React Native agent, your team can monitor your hybrid app's performance and identify any errors within your code. Our agent collects crash data, network traffic, and other information for hybrid apps using native components. This allows your team to better understand the health of your app. We empower your team to make more informed development choices by providing insight into the following:
+
+  * Capture JavaScript errors
+  * Network instrumentation
+  * Distributed tracing (currently functional in Android, iOS support coming in the near future)
+  * Tracking console log, warn and error
+  * Promise rejection tracking
+  * Capture interactions and the sequence in which they were created
+  * Pass user information to New Relic to track user sessions
+  * Expo support
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id=""
+    title="Requirements"
+  >
+Before you install the React Native agent, ensure your app meets these requirements:
+
+  * Android API 21 or higher
```

Comment:
it may be obvious to mobile devs, but i'm a little confused about what the relationship is between the react native agent and the other agents we point to here. i'm gathering they have to have a React Native app of certain version, and a React Native app can be either iOS or Android in nature, and depending on which they have, they have to also meet all the requirements for using our iOS or Android agent. But if my understanding seems wrong, might point to an issue.